.circleci/config.yml

```diff
@@ -317,12 +325,14 @@ jobs:
             - run:
                 name: "Run IPv4 E2E tests"
                 command: |
+                  echo "DEBUGING VAR $E2E_PKG_LIST"
+                  echo "DEBUGING VAR GINKGO_E2E_FLAGS"
                   export CI_K3S_VERSION=<< parameters.k3sVersion >>
-                  export E2E_PKG_LIST=$(circleci tests glob ./test/e2e/* | circleci tests split | xargs printf "./%s/... ")
+                  export E2E_PKG_LIST=${E2E_PKG_LIST:-$(circleci tests glob ./test/e2e/* | circleci tests split | xargs printf "./%s/... ")}
                   export API_VERSION=<< parameters.api >>
                   export GINKGO_TEST_RESULTS_DIR=/tmp/ginkgo-test-results
                   export K3D=true
-                  make -j2 test/e2e
+                  make -j2 << parameters.target >>
```

Comment:
Not for this one but collapsing ipv6 and ipv4 tests should be possible I believe